### ISSUE TITLE
[🐞] Qwik City app in pnpm monorepo fails to load QRLs

### ISSUE THREAD

**Issue by osdiab** [QwikDev/qwik#6656]
### Which component is affected?

Qwik City (routing)

### Describe the bug

When I have an app as a standalone project, things seem to work fine. But when I put that project into a pnpm monorepo, it seems to fail to load QRLs.

Maybe this has to do with `pnpm` hoisting dependencies to the monorepo root?

### Reproduction

https://github.com/osdiab/repro-qwik-ui-use-popover-error

### Steps to reproduce

- pull down that repo
- run `pnpm --filter test-web dev`
- load the page, you'll see some text that says `Hover Me`
- hover it, you get errors in the console and failures in the network.
- run `git checkout 4f5f73e48bcf259bba388bd768b7ea5cd7a08449` to get the previous commit
- run `pnpm dev`
- hover it, it works fine

### System Info

```shell
System:
    OS: macOS 14.5
    CPU: (10) arm64 Apple M1 Max
    Memory: 70.27 MB / 64.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.7.3 - ~/.asdf/installs/nodejs/21.7.3/bin/node
    Yarn: 1.22.19 - ~/.asdf/installs/nodejs/21.7.3/bin/yarn
    npm: 10.5.0 - ~/.asdf/plugins/nodejs/shims/npm
    pnpm: 8.6.2 - ~/.asdf/installs/nodejs/21.7.3/bin/pnpm
    Watchman: 2024.05.06.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 126.0.6478.127
    Edge: 126.0.2592.87
    Safari: 17.5
  npmPackages:
    @builder.io/qwik: ^1.6.0 => 1.7.0
    @builder.io/qwik-city: ^1.6.0 => 1.7.0
    typescript: 5.4.5 => 5.4.5
    undici: * => 6.19.2
    vite: ^5.2.10 => 5.3.3
```

### Additional Information

made a video to show what I mean, idk why it did a 504 this time but it gives me 404's in general.

https://www.loom.com/share/d7a92c93a41c44e39350362d13016254

**Comment by gioboa** [comment]
Thanks for the report, this is fixed with Qwik 1.7.0.
Can you give it a try please?

**Comment by osdiab** [comment]
it worked, nice!
